eZ/Bundle/EzPublishRestBundle/Resources/config/input_parsers.yml

```diff
@@ -67,6 +67,16 @@ parameters:
 
     ezpublish_rest.input.parser.internal.sortclause.data_key_value_object.class: eZ\Publish\Core\REST\Server\Input\Parser\SortClause\DataKeyValueObjectClass
 
+    ezpublish_rest.input.parser.internal.facetbuilder.content_type.class: eZ\Publish\Core\REST\Server\Input\Parser\FacetBuilder\ContentTypeParser
```

Comment:
We do reuse them sometimes (in case of factories, or services that uses same implementation but are not aliases to each other).

I'm for leaving it as it is now, because it's cleaner not to mix both ways and we can't remove previous parameters because of BC break.